reorder-journal.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+if ! [ -x "$(command -v hledger)" ]; then
+  echo 'Error: hledger is not installed.' >&2
+  exit 1
+fi
+
+LEDGER_FILE="$1"
+
+if [[ -z "$LEDGER_FILE" ]]; then
+  echo "Please specify the ledger file in the first argument. ./reorder-journal.sh myledger.journal"
+  exit 1
+fi
+
+TMPFILE_SORTED=$(mktemp /tmp/hledger.XXXXXX)
+TMPFILE_SKELETON=$(mktemp /tmp/hledger.XXXXXX)
+
+hledger print -f "$LEDGER_FILE" > "$TMPFILE_SORTED"
+
+sed '/;;;;;;;;;;;;;;;;;;;;;;;;;;;;;/q' "$LEDGER_FILE" > "$TMPFILE_SKELETON"
```

Comment:
As follow up, lets return early an error in case  `;;;;;;;;;;;;;;;;;;;;;;;;;;;;;` doesn't appear (or appears several times) in the file

Reply:
@AronNovak ^^ (I'll add you as a committer to the repo for easier branching)